internal/client.go

```diff
@@ -578,6 +581,7 @@ func NewClient(service workflowserviceclient.Interface, domain string, options *
 	if options != nil && options.Authorization != nil {
 		service = auth.NewWorkflowServiceWrapper(service, options.Authorization)
 	}
+	service = isolationgroup.NewWorkflowServiceWrapper(service, options.IsolationGroup)
```

Comment:
check if `options.IsolationGroup` is empty string?

Reply:
good call